mcmd/commands/delete.py

```diff
@@ -98,15 +98,15 @@ def _delete_package_contents(args):
 
 def _delete_entity_types_in_package(package_id):
     response = client.get(
-        config.api('rest2') + ResourceType.ENTITY_TYPE.get_entity_id() + '?attrs=id&q=package==' + package_id)
+        api.rest2() + ResourceType.ENTITY_TYPE.get_entity_id() + '?attrs=id&q=package==' + package_id)
```

Comment:
- ```package_id``` should be RSQL encoded (as described in https://github.com/jirutka/rsql-parser)
- ```package_id``` should be encoded (something like https://stackoverflow.com/a/51649797)